### ISSUE TITLE
expand user ~

### ISSUE THREAD

**Issue by reinzor** [Rayman/ros-get#27]
specifying a ros-distro-url with a tilde is not being expanded

**Comment by reinzor** [comment]
Or, we could enforce file://

**Comment by reinzor** [comment]
And we should check for `file://` or `http(s)://`

**Comment by Rayman** [comment]
What command did you run?

**Comment by reinzor** [comment]
Ws create

**Comment by Rayman** [comment]
You could have an URL with a tilde in it: http://example.com/~rayman/index.yaml do I don't think this feature belongs in `ros-get`.

If you want, you could use `$HOME` in your query which bash replaces just before the arguments go to `ros-get`.

**Comment by reinzor** [comment]
Argument taken. But I think it is still nice to check for a valid url. Maybe only expanduser if the scheme is file:
```
from urllib2 improt urlparse
parsed_url = urlparse.urlparse(url)
if not parsed_url.scheme:
    print "Invalid scheme"
if parsed_url.scheme == "file":
    # Expand user here 
```

**Comment by Rayman** [comment]
:+1:

**Comment by reinzor** [comment]
Using $HOME in your url is not working btw

**Comment by Rayman** [comment]
For me it seems to work:
```
$ ros-get ws-create file://$HOME/ros/kinetic/somefile /opt/ros/kinetic/
$ cat .ros-get/rosdistro_index_url
file:///home/username/ros/kinetic/somefile
```

**Comment by reinzor** [comment]
I would like to check for validity, like we discussed last Tuesday

**Comment by reinzor** [comment]
Oh, I see this has been solved in #37, nvm
